### PR TITLE
Full web

### PR DESCRIPTION
**Pull request by George7h**
In this PR the following changes were made:

- Create a website with 3 pages: Home, Calculator, and Quote.
- For every page, you should create a <Route>.
- Make sure the nav links are displayed
- Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/routes/CalculatorComponent.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line
+import Calculator from '../components/Calculator';
```
Comment (prolajumokeoni) [inline src/routes/CalculatorComponent.js]: - [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿 


```suggestion

import Calculator from '../components/calculator';
```

- [x] Please update your import to the correct path,  there is an error in the browser the first letter in calculator is not capitalized 👍🏿

File: src/routes/GetQuoteComponent.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line
+import GetQuote from '../components/GetQuote';
```
Comment (prolajumokeoni) [inline src/routes/GetQuoteComponent.js]: - [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿 


```suggestion
import GetQuote from '../components/getQuote';
```

- [x] Please update your import to the correct path,  there is an error in the browser the first letter in getQuote is not capitalized 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @George7h 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

  - Your website consists of Home, Calculator, and Quotes pages 🚀 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/CalculatorComponent.js (hunk 1 above)
Comment (George7h) [reply src/routes/CalculatorComponent.js]: Hi @prolajumokeoni ,

Thank you for the review. However, the path was referring to "Calculator"- Capitalized as it should be. Your suggestion says that it's not capitalized but it is and the inline suggestions suggest that it should be in lowercase.  The code as is even if removing the disable eslint line works completely in the VSCODE no errors are being recorded. However, Pushing it to the repo it has a eslint error that suggests adding.js file extension to the referred to file path. I have reinstalled the modules etc you can see in the commit history, but no such error occurs in my VScode or browser. See the snapshot attached 
image
And by adding the Github (online) eslint suggestion the whole site doenst work
 Any suggestions?

File: src/routes/GetQuoteComponent.js (hunk 2 above)
Comment (George7h) [reply src/routes/GetQuoteComponent.js]: Hi @prolajumokeoni ,

Thank you for the review. However, the path was referring to "GetQuote"- Capitalized as it should be. Your suggestion says that it's not capitalized but it is and the inline suggestions suggest that it should be in lowercase.  The code as is even if removing the disable eslint line works completely in the VSCODE no errors are being recorded. However, Pushing it to the repo it has a eslint error that suggests adding.js file extension to the referred to file path.  I have reinstalled the modules etc you can see in the commit history, but no such error occurs in my VScode or browser. See the snapshot attached 
image And by adding the Github (online) eslint suggestion the whole site doenst work
 Any suggestions?

Comment (George7h) [reply src/routes/GetQuoteComponent.js]: Hi @prolajumokeoni ,

I have made the changes. For some reason in my VScode, the file names etc were correct, however, on Github the file names were not capitalized and didn't bring in the file name change. Thank you for steering me in the right direction.

File: src/routes/CalculatorComponent.js (hunk 1 above)
Comment (George7h) [reply src/routes/CalculatorComponent.js]: Hi @prolajumokeoni ,

I have made the changes. For some reason in my VScode, the file names etc were correct, however, on Github the file names were not capitalized and didn't bring in the file name change. Thank you for steering me in the right direction

File: src/routes/GetQuoteComponent.js (hunk 2 above)
Comment (prolajumokeoni) [reply src/routes/GetQuoteComponent.js]: Hi @George7h, all the best with your next review, good job working on the requested changes 👍🏿

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @George7h👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with routes. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**